web/src/condense.js

```diff
@@ -118,7 +118,7 @@ export function toggle_collapse(message) {
     // * If the message is fully visible, either because it's too short to
     //   condense or because it's already uncondensed, collapse it
 
-    const $row = message_lists.current.get_row(message.id);
+    const $row = message_lists.current?.get_row(message.id);
     if (!$row) {
         return;
     }
```

Comment:
Hmm this block also seems like a case for an assertion; they literally must have clicked or used a hotkey with this message in view, so it shouldn't be possible for it to not be in the current message list, right?

Reply:
Yeah, changed.